{{cookiecutter.project_slug}}/clients/mobile/react-native/README.md

```diff
@@ -172,7 +174,7 @@ We currently use `expo update` when building our staging app to get a quick and
 There are certain situations when this may not be possible for example we are installing a package that does not currently have an expo extension (revenue cat for in-app purchases) or we are using a native package that expo does not have access to (face id)
 
 
-When mergning into main we deploy a new staging version that can be run in expo we also build a staging version of the app as a stand-alone native build that can be ran on a device. Staging versions will point to he staging backend defined in the [eas.json](./eas.json)
+When merging into main we deploy a new staging version that can be run in expo we also build a staging version of the app as a stand-alone native build that can be ran on a device. Staging versions will point to he staging backend defined in the [eas.json](./eas.json)
```

Comment:
Fixed the order of steps here a little. Added some extra clarity. Fixed spelling